packages/manager/apps/pci-kubernetes/src/pages/oidc-provider/AddOIDCProvider.page.tsx

```diff
@@ -0,0 +1,242 @@
+import {
+  OsdsButton,
+  OsdsFormField,
+  OsdsInput,
+  OsdsModal,
+  OsdsSpinner,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { Translation, useTranslation } from 'react-i18next';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_INPUT_TYPE,
+  ODS_SPINNER_SIZE,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { useNotifications } from '@ovhcloud/manager-components';
+import { useAddOidcProvider } from '@/api/hooks/useKubernetes';
+
+export default function AddOIDCProvider() {
+  const { t } = useTranslation('add-oidc-provider');
+  const { t: tCommon } = useTranslation('common');
+  const { projectId, kubeId } = useParams();
+  const navigate = useNavigate();
+  const onClose = () => navigate('..');
+  const [formState, setFormState] = useState({
+    issuerUrl: '',
+    issuerUrlError: false,
+    issuerUrlTouched: false,
+    clientId: '',
+    clientIdError: false,
+    clientIdTouched: false,
+  });
+  const { addError, addSuccess } = useNotifications();
+  useEffect(() => {
+    setFormState({
+      ...formState,
+      clientIdError: formState.clientIdTouched && !formState.clientId,
+      issuerUrlError: formState.issuerUrlTouched && !formState.issuerUrl,
+    });
+  }, [
+    formState.clientId,
+    formState.clientIdTouched,
+    formState.issuerUrl,
+    formState.issuerUrlTouched,
+  ]);
```

Comment:
```suggestion
const hasClientIdError = formState.clientIdTouched && !formState.clientId;
const hasIssuerUrlError: formState.issuerUrlTouched && !formState.issuerUrl;
```
i don't think an useEffect and storing errors in formState is needed here as you can derive error state from the formState